test/Test.cpp

```diff
@@ -619,6 +676,16 @@ TEST(cli, cmd_delete)
     REQUEST("role");
     CHECK_RETCODE(CMDLINE_RETCODE_COMMAND_NOT_FOUND);
 }
+TEST(cli, cmd_escape)
+{
+  INIT_BUF();
+  REQUEST("echo \\\"");
+  ARRAY_CMP(RESPONSE("\\\" "), buf);
+
+  INIT_BUF();
+  REQUEST("echo \"\\\\\"\"");
+  ARRAY_CMP(RESPONSE("\\\" "), buf);
```

Comment:
```
/>echo \"\\\\\"\" 
\"\\\\\"\" 
retcode: 0
```

Reply:
there are more tests which get different result locally. but that probably because `set` wasn't in the binary.

Reply:
note: in C \ will escape next char -> you can reproduce this manually by writing: `echo ”\\\”`

Reply:
what abou these? when echo escape 
```
/>echo "\\\" 
"\\\" 
retcode: 0
/>echo "\\\\"" 
\\" 
retcode: 0
/>echo "\\\\"\" 
"\\\\"\" 
retcode: 0
/>echo "\\\"" 
\" 
retcode: 0
/>echo "\\"" 
\" 
```

Reply:
`/>echo "\\\\"\"`

I would say that this can be known issue for now since it doesn't give much benefits even spend some time to fix and easy "workaround" to achieve same results is:
```
/>echo "\\\\\"" 
\\"
``` 
which are more reasonable in my mind..

Reply:
agree